.eslintrc

```diff
@@ -38,7 +38,9 @@
     // The beauty of TS is that it infers types quite well, so let's not write
     // too much code.
     "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/no-explicit-any": "error"
+    "@typescript-eslint/no-explicit-any": "error",
+    // Disable camelcase rule
+    "camelcase": "off"
```

Comment:
This could be part of our shared config, no?

Reply:
Yes, but my goal was to try to get the linter passing, which was simpler to do by just adding it here. Once everything is ready to go (which might not be soon), I will either remove it and update `eslint-config-amo` with the rule, or we could also land this and then file an issue to update `eslint-config-amo` and remove it from here.

Reply:
oh yeah, that was not a change request, just a thought.